src/index.ts

```diff
@@ -0,0 +1,140 @@
+import { SecretManagerServiceClient, protos } from "@google-cloud/secret-manager";
+
+/**
+ * Filter function to filter secrets by name that dont have to be loaded
+ */
+type FilterFunction = (params: {
+  /**
+   * The name of the secret (without the path)
+   */
+  name: string;
+
+  /**
+   * The secret itself
+   */
+  secret: protos.google.cloud.secretmanager.v1.ISecret;
+}) => boolean;
+
+/**
+ * Definition of possible options for the module
+ */
+type WithGoogleSecretsOptions = {
+  /**
+   * The project name in google could to load the secrets from
+   */
+  projectName: string;
+
+  /**
+   * A mapping to define which secret should overwritte which config entry (secretName, configEntry(ies))
+   */
+  mapping: Record<string, string | string[]>;
+
+  /**
+   * The filter to filter directly by labels in gcp
```

Comment:
@neotob 

It's not only for filtering by labels, thats more an example. Can you just link to the filter like in the readme? thanks!

Reply:
oh i wasn't aware of this :D removed the label hint and added link to gcp